docs/2023/ABDM/Frontend And Deep Linking/index.md

```diff
@@ -0,0 +1,19 @@
+---
+sidebar_position: 1
+title: Introduction
+slug: /2023/ABDM/Data Transfer Workflow
+---
+
+
+# Author
+Aman Gadadare
+
+# Contact Info
+ Email  amangadadare2000@gmail.com
+
+Linked In()  https://www.linkedin.com/in/aman-gadadare-589918195/
+
+GitHub()  https://github.com/AmanGadadare
```

Comment:
The folder name should be name of your project. Contact information is not in the appropriate format.